pages/concepts/drop-in-components.mdx

```diff
@@ -135,7 +136,7 @@ dwolla-error-message
 
 ### Create an Unverified Customer
 
-The `dwolla-customer-create` component is used to display a form to collect the required information needed to create an Unverified Customer. An Unverified Customer is a customer type that requires a minimal amount of information in order to be created. This information includes: First Name, Last Name, Email, Optionally: Business Name for Businesses. To find out more about the abilities and limitations of this customer type, [visit our docs](/concepts/customer-types) on concepts.
+The `dwolla-customer-create` component is used to display a form to collect the required information needed to create an Unverified Customer. An Unverified Customer is a customer type that requires a minimal amount of information in order to be created. This information includes: First Name, Last Name, Email, Optionally: Business Name for Businesses. To allow users to enter a Business Name, include the optional attribute `isBusineses` in the drop-in component. To find out more about the abilities and limitations of this customer type, [visit our docs](/concepts/customer-types) on concepts.
```

Comment:
`isBusineses` should be `isBusiness`

Reply:
Oh good catch! [Here's a fix](https://github.com/Dwolla/dev-portal/pull/337/commits/2b949584b0d66a79256137b519cb3c152c416e1f)!